src/components/Player/Pages/MMR/MMR.jsx

```diff
@@ -17,7 +26,7 @@ const MMRInfo = (
 const MMR = ({ columns, error, loading }) => (
   <div>
     <Container title={strings.heading_mmr} subtitle={MMRInfo} error={error} loading={loading}>
-      <MMRGraph columns={columns} />
+      <MMRGraph columns={filterNullValues(columns)} />
```

Comment:
actually investigate inside the mmrgraph method, it may make more sense to filter in there.